src/scripts/Script1.fsx

```diff
@@ -0,0 +1,45 @@
+﻿
+#r @"D:\Gibran\Projetos\GitRep\SQLProvider\bin\FSharp.Data.SqlProvider.dll"
+open FSharp.Data.Sql
+open FSharp.Data.Sql.Providers
+
+
+let [<Literal>] resolutionPath = "D:/Gibran/Projetos/GitRep/SQLProvider/packages/scripts/FirebirdSql.Data.FirebirdClient/lib/net452" //"D:/Gibran/Projetos/FirebirdNetProvider141216/src/FirebirdSql.Data.FirebirdClient/bin/Debug/NET40" 
+let [<Literal>] connectionString = @"character set=NONE;data source=localhost;port=3051;initial catalog=d:\Tisul\Gestao\Dados\ROMENA.FDB;user id=SYSDBA;password=masterkey;dialect=1"
+// create a type alias with the connection string and database vendor settings
+
+              
+
+
+
+
+type sql = SqlDataProvider< 
```

Comment:
Currently our test cases are at `/SQLProvider/tests/SqlProvider.Tests/scripts` by the name of the provider and file is included in SqlProvider.Tests.fsproj. Needed sample database scripts are at `src/DatabaseScripts`